packer/github-actions-runner/variables.pkr.hcl

```diff
@@ -7,7 +7,7 @@ variable "region" {
 variable "instance_type" {
   description = "The instance type Packer will use for the builder"
   type        = string
-  default     = "t3.large"
+  default     = "t3.xlarge"
```

Comment:
This change can be dropped. The size that packer uses for building the image doesn't affect the size of the instances used for running workflows.